packages/server-wallet/server-wallet.api.md

```diff
@@ -3,101 +3,101 @@
 > Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
```

Comment:
The changes here are only formatting changes. But we still need them. It's an auto-generated file.